client/state/analytics/reducer.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import {
+	ANALYTICS_CONTINOUS_MONITOR_ON,
+	ANALYTICS_CONTINOUS_MONITOR_OFF
+} from 'state/action-types';
+
+export const isTracking = ( state = false, { type } ) => {
```

Comment:
Furthermore, since the corresponding action accepts a tracker arg, we should probably modify the reducer to keep track of that too. Something like

```js
export const isTracking = ( state = {}, { type, payload: trackingTool } ) => {
	if ( ANALYTICS_CONTINOUS_MONITOR_ON === type ) {
		return { ...state, trackingTool: true };
	}

	if ( ANALYTICS_CONTINOUS_MONITOR_OFF === type ) {
		return { ...state, trackingTool: false };
	}

	return state;
};
```

This changes `state` to an object that has tracking tool names as its keys, and booleans indicating whether that tool is running or not as values.